crates/router/src/core/fraud_check/operation/fraud_check_post.rs

```diff
@@ -177,45 +182,54 @@ impl<F: Send + Clone> Domain<F> for FraudCheckPost {
     async fn execute_post_tasks(
         &self,
         state: &AppState,
+        req_state: ReqState,
         frm_data: &mut FrmData,
         merchant_account: &domain::MerchantAccount,
         frm_configs: FrmConfigsObject,
         frm_suggestion: &mut Option<FrmSuggestion>,
         key_store: domain::MerchantKeyStore,
         payment_data: &mut payments::PaymentData<F>,
         customer: &Option<domain::Customer>,
+        _should_continue_capture: &mut bool,
     ) -> RouterResult<Option<FrmData>> {
         if matches!(frm_data.fraud_check.frm_status, FraudCheckStatus::Fraud)
-            && matches!(frm_configs.frm_action, FrmAction::AutoRefund)
+            && matches!(frm_configs.frm_action, FrmAction::CancelTxn)
```

Comment:
We had three frm configs right? 
1. Pre frm -> Cancel on fraud
2. Pre frm -> Complete transaction and mark as fraud. merchant will trigger refund if needed
3. Post frm -> Auto refund on fraud 
4. Post frm -> Authorize the payment and send it for manual review. Cancel or Capture payment based on merchant decision

Why are we removing the 3 option here?

Reply:
In Revised flow, We don't actually capture the payment rather we authorize the payments(Post Frm) then take a call whether to cancel/hold/capture . When merchant approves ,we capture the payment or else we cancel the payment.